command/ca/token.go

```diff
@@ -490,6 +500,8 @@ func provisionerPrompt(ctx *cli.Context, provisioners provisioner.List) (provisi
 			return true
 		case provisioner.TypeGCP, provisioner.TypeAWS, provisioner.TypeAzure:
 			return true
+		case provisioner.TypeACME:
+			return ctx.Command.Name == "certificate"
```

Comment:
Do we want to support `step provisioner sign`?

Reply:
assuming you mean `step ca sign`